topics/single-cell/tutorials/scrna-umis/tutorial.md

```diff
@@ -43,7 +43,7 @@ recordings:
 
 Barcodes are small oligonucleotides that are inserted into the captured sequence at a specific point, and provide two pieces of information about the sequence:
 
- 1. Which cell the sequence came from
+ 1. Which cell the sequence came from?
```

Comment:
reading the sentence before it, maybe it should just be a period at the end of both items, what do you think?

Reply:
or, if you want it to be a question, rephrase it "Which cell **did** the sequence **come** from?"

Reply:
Thanks @shiltemann-agree with the latter point! Tbh, we were simply testing the system by making minor changes.

Reply:
Ace - try committing the change on this branch to make both sentences questions! (Does that make sense? Select 'Files changed' above and you should be able to then Edit the file. Commit that change)
Then we can review and approve and you'll have experienced a true Github collaboration!